MOLFCMClient/MOLFCMClient.m

```diff
@@ -82,19 +91,23 @@ @implementation MOLFCMClient
 #pragma mark init/dealloc methods
 
 - (instancetype)initWithFCMToken:(NSString *)FCMToken
+                            host:(NSString *)host
+                 connectDelayMax:(uint32_t)connectDelayMax
+                      backoffMax:(uint32_t)backoffMax
+                      fatalCodes:(NSArray<NSNumber *> *)fatalCodes
             sessionConfiguration:(NSURLSessionConfiguration *)sessionConfiguration
                   messageHandler:(MOLFCMMessageHandler)messageHandler {
   self = [super init];
   if (self) {
     _FCMToken = [FCMToken copy];
-    _bindComponents = [NSURLComponents componentsWithString:kFCMHost];
+    _bindComponents = [NSURLComponents componentsWithString:host ?: kFCMHost];
```

Comment:
host is not marked nonnull, passing nil will throw an exception.

Reply:
Don't think so elvis.

Reply:
Oh oops.